ra/ra.go

```diff
@@ -241,6 +242,12 @@ func NewRegistrationAuthorityImpl(
 	})
 	stats.MustRegister(certCSRMismatch)
 
+	pauseCounter := prometheus.NewCounterVec(prometheus.CounterOpts{
+		Name: "paused_pairs",
+		Help: "Number of times a pause operation is performed, labeled by paused=[bool], repaused=[bool], grace=[bool]",
```

Comment:
Perhaps it's too much for the help string here, but it would be good to document what `repaused` and `grace` mean. Perhaps in a comment either here or where they are incremented?

Alternately, we could remove `paused=[bool], repaused=[bool], grace=[bool]` from the help string since that information is available direct from Prometheus, and use some of the saved space to explain the two less obvious labels.